### PR TITLE
Add commit format rules

### LINKED ISSUE #485: Add commit message rules

### PR DESCRIPTION
**Pull request by kantord**
fixes #485

Do not squash<br/><br/><br/><url>LiveURL: https://smartfaq-add-commit-format-rules.surge.sh</url>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,6 +80,7 @@
     "graphql-cli": "2.15.8",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "3.0.6",
+    "husky": "^0.14.3",
```
Comment (jaroslav-kubicek) [inline package.json]: why this? :)

Comment (kantord) [reply package.json]: It's used to set up git hooks from `package.json`. It's the official suggestion from commitlint's documentation. Here's the git hook config that husky is used for https://github.com/kiwicom/smart-faq/blob/bc04ee40c20f35602d1e83b22a471cff04c08e4d/package.json#L33

**Comment by kantord** [comment]
> Pls add some docs what's expected from developers.

Good point!

File: package.json [hunk 2]
```diff
@@ -29,11 +29,14 @@
     "translations:collect": "i18next-scanner 'src/**/*.{js,jsx}'",
     "translations:check": "node scripts/checkTranslations.js",
     "relay": "relay-compiler --src ./src --schema ./schema.graphql",
-    "schema": "graphql get-schema dev"
+    "schema": "graphql get-schema dev",
+    "commitmsg": "commitlint -E GIT_PARAMS"
```
Comment (dinodsaurus) [inline package.json]: This is a pre-commit hook?

Comment (kantord) [reply package.json]: Yes and also merge https://git-scm.com/docs/githooks#_commit_msg

Comment (kantord) [reply package.json]: Recommended by commitlint documentation here http://marionebl.github.io/commitlint/#/guides-local-setup?id=install-husky

**Pull request merged**